packages/front/svelte/views/auth/forget-pwd/ForgetPwd.svelte

```diff
@@ -9,9 +11,10 @@
     const { email, promise, firstSubmitted } = ctrl;
 </script>
 
-<div class="fr-grid-row fr-grid-row--center fr-grid-row--gutters">
-    <div class="fr-col fr-col-lg-8">
-        <h1>Mot de passe perdu</h1>
+<div class="fr-grid-row fr-grid-row--center fr-p-8v">
+    <div class="forget-pwd fr-col-6 fr-p-8v">
+        <div class="svg">{@html padlockSvg}</div>
+        <h2 class="fr-mt-6v">Mot de passe oublié ?</h2>
```

Comment:
```suggestion
        <h1 class="fr-h2 fr-mt-6v">Mot de passe oublié ?</h2>
```
enjeu d'accessibilité et de sémantique de la page

Reply:
ça change quoi ? tu pars du principe que vu que c'est le seul titre ça devrait être un h1 ? Moi je suis le design (qui dit h2)

Reply:
Exactement, d'un point de vue sémantique html le titre de page doit être un h1. La classe `fr-h2` est là pour respecter le design en même temps. 
Il faut aussi ne pas faire de saut (h1 -> h5 par exemple) mais ce n'est pas l'enjeu ici

Reply:
Je trouve qu'il est souvant difficile de respercter cette sémantique par exemple sur la page d'accueil (la recherche) on à pas vraiment de titre, on va pas mettre un h1 sur "Vos dernières recherches" non ?

Reply:
Dans ce cas faut faire une grosse passe sur toute l'application, à mon avis on ne respecte jamais cette régle

Reply:
On peut déjà la respecter sur les nouvelles pages, mais effectivement pas ouf. Je fais un ticket